src/main/java/service/ReportCreator.java

```diff
@@ -0,0 +1,25 @@
+package service;
+
+import db.Storage;
+
+public class ReportCreator {
+
+    private static final String DATA_SEPARATOR = ",";
+    private static final StringBuilder storeActivitiesReport = new StringBuilder();
+
+    public static String createReport(Storage storage) {
+
+        storeActivitiesReport.append("fruit,quantity")
```

Comment:
Please extract this header to a constant :)

Reply:
Thanks for your advice, have fixed that.